app/resources/css-default.template

```diff
@@ -143,7 +143,7 @@ pre {  /* code samples */
   color: white;
   background-color: #616161;
   height: 64px;
-  line-height: 64px; /* vertical center text */
+  line-height: 64px; /* Vertically center the text */
```

Comment:
Add one more space before "/*". I'm not sure if there is CSS style guide, but let's follow the general styling in Google to put two whitespaces before inline comment.